src/test/java/uk/gov/pay/ledger/queue/QueueMessageReceiverIT.java

```diff
@@ -155,4 +155,34 @@ public void shouldHandleRefundAvailabilityUpdatedEvent() throws InterruptedExcep
                 .body("transaction_id", is(resourceExternalId))
                 .body("refund_summary.amount_available", is(200));
     }
+
+    @Test
+    public void shouldHandleForceCaptureEvent() throws InterruptedException {
+        final String resourceExternalId = "rexid2";
+        final String gatewayAccountId = "test_accountId";
+        aQueuePaymentEventFixture()
+                .withResourceExternalId(resourceExternalId)
+                .withEventDate(CREATED_AT)
+                .withEventType("AUTHORISATION_REJECTED")
+                .withEventData(format("{\"gateway_account_id\":\"%s\"}", gatewayAccountId))
+                .insert(rule.getSqsClient());
+
```

Comment:
I think it'll be good to query `"/v1/transaction/" + resourceExternalId + "?account_id=" + gatewayAccountId` here and verify the status is `FAILED_REJECTED` here.
That way when you put the next event type `EXPUNGED_CHARGE_STATUS_CORRECTED_TO_CAPTURED_TO_MATCH_GATEWAY_STATUS` on the queue we are confident the status was actually changed to `success`.

Reply:
Done